src/_assets/javascripts/components/moving-allowances-table.js

```diff
@@ -6,13 +6,27 @@
 
     $window.on('update', this.events.update.bind(this));
 
+    this.$el.find('.view-details').each(function() {
+      var $clone = $(this.hash).clone();
+
+      $clone.find('h4, .return-to-moving-allowances').remove();
```

Comment:
Removing some content from the cloned node that doesn't make sense when displayed inside the `<table>`.